src/components/Filters/Filters.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { useState } from 'react';
+import styles from '../../common/commonOptionButton.module.css';
+import filterStyles from './Filters.module.css';
+
+const filterOptions = ['Animals', 'Health', 'Education', 'Environment'];
+
```

Comment:
Why in ```SortByButton``` you declared the type of the options and here not? (yes I am annoying lol)